Dfe.Academies.Academisation.Domain.Core/ApplicationAggregate/SchoolDetails.cs

```diff
@@ -30,5 +30,8 @@ public record SchoolDetails(
 	int? ProjectedPupilNumbersYear2 = null,
 	int? ProjectedPupilNumbersYear3 = null,
 	string? CapacityAssumptions = null,
-	int? CapacityPublishedAdmissionsNumber = null
+	int? CapacityPublishedAdmissionsNumber = null,
+	// application pre-support grant
+	string? SchoolSupportGrantFundsPaidTo = null,
```

Comment:
If there are only two possible answers to this, should it be an enum?

Reply:
possibly, but following the existing pattern in place which doesn't have an enum for applicationType / applicationStatus / contributor role .

Reply:
yeah, although given that we're rewriting the application from scratch, will there ever be a better time to make this change? We could still map to the string in the ApplicationSchoolServiceModel so that the front end doesn't have to be changed...